src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  height: 406px;
+  width: 198px;
+  border-radius: 5px;
+  background: #{$background-main};
+  border: 1px solid #{$gray-elements};
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px 40px;
+  }
+  &__container {
+    padding: 0 16px 16px;
+    width: fit-content;
+    &-title {
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      color: #{$main-accent};
+      margin-bottom: 4px;
+    }
+    &-product-code {
+      display: block;
+      margin-bottom: 16px;
+      font-size: 10px;
+      line-height: 14px;
+      color: #{$secondary};
+    }
+    &-rating {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+    }
+    &-reviews {
+      display: block;
+      font-size: 10px;
+      line-height: 14px;
+      color: #{$main-accent};
+    }
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      margin-bottom: 16px;
+      align-items: center;
+      &-title {
+        font-size: 12px;
+        line-height: 18px;
+        color: #{$secondary};
+      }
+      &-value {
+        font-weight: 700;
+        font-size: 16px;
+        line-height: 18px;
+        color: #{$main-accent};
+      }
+    }
+    &-buy-button {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-bottom: 16px;
```

Comment:
'margin-bottom: 16px;' is duplicated it few times, please move it to the variable